DataLink.tex

```diff
@@ -622,10 +622,6 @@ \subsubsection{semantics}
 this term is used to describe links available for the retrieval of the
 file(s) making up what ID references.
 
-Since NULL values are not permitted in the semantics column, when only
-an error\_message is supplied its value \rfcshould\ be the most appropriate
-for the link the service was trying to generate.
-
```

Comment:
I don't know that requiring non-NULL semantics is a good idea, but as long as it's in place, I'd say this guidance for what to do there in case of an error response is useful.  Why remove it?